---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: false
---

[Generation of lineage-resolved complete metagenome-assembled 
genomes by precision phasing](https://doi.org/10.1101/2021.05.04.442591). 
_Nature Biotechnology_, in press (2021)
>Derek M. Bickhart\*, __Mikhail Kolmogorov\*__, Elizabeth Tseng, Daniel Portik, Anton
Korobeynikov, Ivan Tolstoganov, Gherman Uritskiy, Ivan Liachko, Shawn T. Sullivan,
Sung Bong Shin, Alvah Zorea, Victòria Pascal Andreu, Kevin Panke-Buisse, Marnix H. Medema, 
Itzik Mizrahi, Pavel Pevzner & Tim Smith

<hr>

[Assembling viruses and identifying their hosts from long-read metagenomics data](https://github.com/Dmitry-Antipov/viralFlye). 
_Genome Research_, in press (2021)  
>Dmitry Antipov, Mikhail Rayko, __Mikhail Kolmogorov__ & Pavel Pevzner

<hr>

[Haplotype-aware variant calling with PEPPER-Margin-DeepVariant enables high accuracy in nanopore long-reads](https://www.nature.com/articles/s41592-021-01299-w).
_Nature Methods_ (18) 1322–1332 (2021)  
>Shafin, K., Pesout, T., Chang, P.C., Nattestad, M., Kolesnikov, A., Goel, S., Baid, G.,
__Kolmogorov, M.__, Eizenga, J.M., Miga, K.H., Carnevali, P., Jain, M., Carroll, A. & Benedict Paten

<hr>

[Critical Assessment of Metagenome Interpretation - the second round of challenges](https://doi.org/10.1101/2021.07.12.451567). _bioRxiv_ (2021)  
>Meyer, F., Fritz, A., Deng, Z. L., Koslicki, D., Gurevich, A., Robertson, G., (...),
__Kolmogorov M.__, (...), McHardy, A. C

<hr>

[The complete sequence of a human genome](https://www.nature.com/articles/s41592-021-01299-w). _bioRxiv_ (2021)  
>Nurk, S., Koren, S., Rhie, A., Rautiainen, M., Bzikadze, A. V., Mikheenko, A., (...),
__Kolmogorov. M__, (...), Phillippy, A. M.

<hr>

[Assembling Long and Accurate Reads Using de Bruijn Graphs](https://doi.org/10.1101/2020.12.10.420448). _bioRxiv_ (2021)  
>A Bankevich, A Bzikadze, __M Kolmogorov__, D. Antipov & PA Pevzner.

<hr>

[metaFlye: scalable and accurate long-read metagenome assembler](https://www.nature.com/articles/s41592-020-00971-x), _Nature Methods_ (2020)  
>**Mikhail Kolmogorov**, Derek M. Bickhart, Bahar Behsaz, Alexey Gurevich, Mikhail
Rayko, Sung Bong Shin, Kristen Kuhn, Jeffrey Yuan, Evgeny Polevikov, Timothy P. L. Smith
and Pavel A. Pevzner.

<hr>

[Synteny paths for assembly graphs comparison](http://drops.dagstuhl.de/opus/volltexte/2019/11054/), _WABI_ 24:1-24:14 (2019)  
>Evgeny Polevikov and **Mikhail Kolmogorov**

<hr>

[Assembly of long error-prone reads using repeat graphs](https://www.nature.com/articles/s41587-019-0072-8), _Nature Biotechnology_ 37, 540–546 (2019)  
>**Mikhail Kolmogorov**, Jeffrey Yuan, Yu Lin & Pavel Pevzner

<hr>

[Assembly Graph Browser: interactive visualization of assembly graphs](https://academic.oup.com/bioinformatics/article-abstract/35/18/3476/5306331). 
_Bioinformatics_ 35 (18), 3476-3478 (2019)  
>Alla Mikheenko & **Mikhail Kolmogorov**

<hr>

[Chromosome assembly of large and complex genomes using multiple references](https://genome.cshlp.org/content/28/11/1720.full). 
_Genome Research_ 28 (11), 1720-1732 (2018)  
>**Mikhail Kolmogorov**, Joel Armstrong, Brian J. Raney, Ian Streeter, 
Matthew Dunn, Fengtang Yang, Duncan Odom, Paul Flicek, Thomas Keane, David Thybert, Benedict Paten & Son Pham.

<hr>

[High-quality genome sequences of uncultured microbes by assembly of read clouds](https://www.nature.com/articles/nbt.4266). 
_Nature Biotechnology_ 36, 1067–1075 (2018)  
>Alex Bishara, Eli L Moss, **Mikhail Kolmogorov**, Alma Parada, Ziming Weng, Arend Sidow, Anne E Dekas, Serafim Batzoglou, Ami S Bhatt.

<hr>

[Sixteen diverse laboratory mouse reference genomes define strain-specific haplotypes and novel functional loci](https://www.nature.com/articles/s41588-018-0223-8). 
_Nature Genetics_ 50, 1574–1583 (2018)  
>Jingtao Lilue, Anthony G Doran, Ian T Fiddes, Monica Abrudan, (...), **Mikhail Kolmogorov**, (...), Benedict Paten, Thomas M Keane

<hr>

[Repeat associated mechanisms of genome evolution and function revealed by the Mus caroli and Mus pahari genomes](https://genome.cshlp.org/content/28/4/448.short). 
_Genome Research_ 28 (4), 448-459 (2018)  
>David Thybert, Maša Roller, Fábio CP Navarro, Ian Fiddes, Ian Streeter, Christine Feig, David Martin-Galvez, **Mikhail Kolmogorov**, (...), Paul Flicek

<hr>

[Single-Molecule Protein Identification by Sub-Nanopore Sensors](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005356). 
_PloS Computational Biology_ 13 (5), e1005356 (2017)  
>**Mikhail Kolmogorov**, Eamonn Kennedy, Zhuxin Dong, Gregory Timp and Pavel Pevzner

<hr>

[Assembly of long error-prone reads using de Bruijn graphs](https://www.pnas.org/content/113/52/E8396.short). 
_PNAS_ 113 (52), E8396-E8405, (2016),  
>Yu Lin\*, Jeffrey Yuan\*, __Mikhail Kolmogorov\*__, Max Shen and Pavel Pevzner

<hr>

[SpectroGene: a tool for proteogenomic annotations using top-down spectra](https://pubs.acs.org/doi/abs/10.1021/acs.jproteome.5b00610). 
_Journal of Proteome Research_ 15 (1), 144-151 (2015)  
>**Mikhail Kolmogorov**, Xiaowen Liu and Pavel Pevzner

<hr>

[Ragout - a reference-assisted assembly tool for bacterial genomes](https://academic.oup.com/bioinformatics/article-abstract/30/12/i302/388572). 
_Bioinformatics_ 30 (12), i302-i309 (2014)  
>**Mikhail Kolmogorov**, Brian Raney, Benedict Paten, Son Pham

<hr>

[Sibelia: a scalable and comprehensive synteny block generation tool for closely related microbial genomes](https://link.springer.com/chapter/10.1007/978-3-642-40453-5_17). 
_WABI_ (2013)  
>Ilya Minkin, Anand Patel, **Mikhail Kolmogorov**, Nikolay Vyahhi, Son Pham

