---
layout: archive
title: "Our Software"
permalink: /software/
author_profile: false
redirect_to: "https://ccr.cancer.gov/staff-directory/mikhail-kolmogorov"
---

---

[Flye & metaFlye](https://github.com/fenderglass/Flye)
>Flye is a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies.
It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. 
Flye also has a special mode for metagenome assembly.

---

[HapDup](https://github.com/fenderglass/hapdup) 
>HapDup (haplotype duplicator) is a pipeline to convert a haploid long read assembly into a dual diploid assembly.
The reconstructed haplotypes preserve heterozygous structural variants (in addition to small variants) and are locally phased.


---

[Ragout](https://github.com/fenderglass/Ragout)
>Ragout is a tool for chromosome-level scaffolding using multiple references. 
Given initial assembly fragments (contigs/scaffolds) and one or multiple related references (complete or draft), 
it produces a chromosome-scale assembly (as a set of scaffolds).

---

[AGB](https://github.com/almiheenko/AGB)
>Provides interactive visualization of assembly graphs, a wide range of tuning parameters, and various options for modifying/simplifying the graph.

---

[Asgan](https://github.com/epolevikov/Asgan)
>Asgan is a tool for analysis and comparison of assembly graphs. The tool takes two assembly graphs in the GFA format as input and 
finds the minimum set of homologous sequences (synteny paths) shared between the graphs. 
As output, Asgan produces various statistics and a visualization of the found paths.

---

[dipdiff](https://github.com/fenderglass/dipdiff)
>dipdiff is a simple SV calling package for diploid assemblies

----

[maf2synteny](https://github.com/fenderglass/maf2synteny)
>A tool that postprocesses whole genome alignment (for two or more genomes) and produces coarse-grained synteny blocks. 

